SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialReceiver.cpp

```diff
@@ -387,9 +387,12 @@ void USpatialReceiver::HandleActorAuthority(const Worker_AuthorityChangeOp& Op)
 		return;
 	}
 
-	if (NetDriver->VirtualWorkerTranslator)
+	if (Op.component_id == SpatialConstants::VIRTUAL_WORKER_TRANSLATION_COMPONENT_ID
+		&& Op.authority == WORKER_AUTHORITY_AUTHORITATIVE)
 	{
-		NetDriver->VirtualWorkerTranslator->AuthorityChanged(Op);
+		NetDriver->InitializeVirtualWorkerTranslationManager();
+		check(NetDriver->VirtualWorkerTranslationManager != nullptr);
```

Comment:
this seems unnecessary to me since we're making a call to initialize on the previous line

Reply:
That's fair; I was likely being overcautious.